client/src/workspace/layouts/AppsViewLayout.tsx

```diff
@@ -47,16 +48,20 @@ export const AppsViewLayout: React.FC = () => {
                 <AppIcon name={icon} />
                 {app.definition.notes.label || app.definition.id}
               </div>
-              {userGuideLink && (
-                <a
-                  href={userGuideLink}
-                  target="_blank"
-                  rel="noreferrer"
-                  style={{ marginRight: 10 }}
-                >
-                  View User Guide
-                </a>
-              )}
+              <div style={{ display: 'inline-block' }}> 
```

Comment:
This probably doesn't need to be wrapped in a div. Also, try moving the `<SystemStatusModal />` to the bottom of this component, as a modal is it's own component.